### PR TITLE
Jobs opening in AddWebSolution

### PR DESCRIPTION
**Pull request by Pooja-addweb**

### CODE REVIEW

File: public/data/job-openings-data.json [hunk 1]
```diff
@@ -674,6 +674,39 @@
           "experience": "1 to 2 Years experience"
         }
       ]
-    }
+    },
+        {
```
Comment (JaldeepAsodariya) [inline public/data/job-openings-data.json:678]: Please apply Format Document

**Review by JaldeepAsodariya — CHANGES_REQUESTED** [review]

**Comment by JaldeepAsodariya** [comment]
@Pooja-addweb  Can you please check the requested changes?
Thanks

**Comment by Pooja-addweb** [comment]
Sure, I am checking it.

Thanks & Regards,
[image: Logo] <https://www.addwebsolution.com/> [image: Logo]
Pooja Upadhyay
People Operations Manager
[image: Skype icon] hr.addweb | www.addwebsolution.com
07929705640
*Socialize with us on*
[image: Facebook icon]
<https://www.facebook.com/addwebsolution.pvt.ltd> [image:
LinkedIn icon] <https://www.linkedin.com/company/addwebsolution> [image:
Twitter icon] <https://twitter.com/addwebsolution> [image: Youtbue icon]
<https://www.youtube.com/channel/UCtZHMXcJ8lLW-ikFNKOszlQ> [image: Youtbue
icon] <https://tinyurl.com/y3hjg25j>

**Comment by Pooja-addweb** [comment]
Hello Team,

Required changes are done.

Thanks & Regards,
[image: Logo] <https://www.addwebsolution.com/> [image: Logo]
Pooja Upadhyay
People Operations Manager
[image: Skype icon] hr.addweb | www.addwebsolution.com
07929705640
*Socialize with us on*
[image: Facebook icon]
<https://www.facebook.com/addwebsolution.pvt.ltd> [image:
LinkedIn icon] <https://www.linkedin.com/company/addwebsolution> [image:
Twitter icon] <https://twitter.com/addwebsolution> [image: Youtbue icon]
<https://www.youtube.com/channel/UCtZHMXcJ8lLW-ikFNKOszlQ> [image: Youtbue
icon] <https://tinyurl.com/y3hjg25j>

**Review by JaldeepAsodariya — APPROVED** [review]

**Comment by Pooja-addweb** [comment]
Hello,

Kindly inform me when I can see my Job live on the site ?.

Thanks & Regards,
[image: Logo] <https://www.addwebsolution.com/> [image: Logo]
Pooja Upadhyay
People Operations Manager
[image: Skype icon] hr.addweb | www.addwebsolution.com
07929705640
*Socialize with us on*
[image: Facebook icon]
<https://www.facebook.com/addwebsolution.pvt.ltd> [image:
LinkedIn icon] <https://www.linkedin.com/company/addwebsolution> [image:
Twitter icon] <https://twitter.com/addwebsolution> [image: Youtbue icon]
<https://www.youtube.com/channel/UCtZHMXcJ8lLW-ikFNKOszlQ> [image: Youtbue
icon] <https://tinyurl.com/y3hjg25j>

**Comment by JaldeepAsodariya** [comment]
@Pooja-addweb It's live now, Please check over the official website.
Thank you

**Pull request merged**